dias/analyzers/source_spectra_analyzer.py

```diff
@@ -0,0 +1,733 @@
+
+    Indexes
+    .......
+    freq
+        1D array of type `float` representing an index of frequencies.
+    pol
+        1D array which references the polarization for the feed.
+        There are two polarizations:
+        index 0 is E-W polarization and
+        index 1 is N-S polarization.
+    time
+        1D array of type `float` that contains the Unix timestamps at which data is recorded.
+    ra
+        1D array which contains the right ascension of the source.
+    ha
+        1D array which contains the hour angle covered in the output file.
+
+    Datasets
+    ........
+    vis
+        Fringestopped visibility as a function of frequency and time.
+    weight
+        Inverse variance, computed from the fast cadence data,
+        as a function of frequency and time.
+    count
+        Normalization to the summed and weighted noise variance.
+
+    Attributes for output dataset
+    .............................
+    instrument_name
+        Correlator for the acquired (chime).
+    collection_server
+        Machine at which the script is run.
+    system_user
+        User account running the script.
+    git_version_tag
+        Version of code used.
+
+    State Data
+    -----------
+    None
+
+    Config Variables
+    -----------------------
+
+    Attributes
+    ----------
+    lag : timedelta
+        Number of hours before time of script execution for searching the files
+    correlator : str
+        Source of the input data
+    acq_suffix : str
+        Type of dataset to be read
+    source_transits : list of str
+        Source tranits that we want analyses for
+    output_suffix : str
+        Suffix for the output file
+    cyl_start_char : int
+        Starting character for the cylinders (ASCII)
+    cyl_start_num : int
+        Offset for CHIME cylinders
+        (due to inital numbers allotted to pathfinder)
+    sep_cyl : bool
+        option to preserve cylinder pairs
+    include_auto : bool
+        option to include autocorrelation
+    include_intracyl : bool
+        option in include intracylinder baselines
+    include_crosspol : bool
+        option to include crosspol data
+    rot : float
+        Rotation angle of the cylinder in degrees
+    nfreq_per_block : int
+        number of frequency channels to be run in one block
+        loading all frequency channels at the same time leads to memory error
+    nsgima : float
+        Span required from the peak of transit
+        in terms of sigma
+    perform_fit : boolean
+        Performs a Gaussian fit to transit
+
+    """
+
+    # Config parameters related to scheduling
+    lag = config.Property(proptype=str2timedelta, default="4h")
+
+    # Config parameters defining data file selection
+    correlator = config.Property(proptype=str, default="chime")
+    acq_suffix = config.Property(proptype=str, default="stack_corr")
+    source_transits = config.Property(proptype=list, default=["cyg_a"])
```

Comment:
We should be processing many more sources than Cyg A.  It may be easier to specify a flux and declination threshold as config parameters and then process all sources in the flux catalog that are above that threshold, rather than listing them by name.